src/App.jsx

```diff
@@ -14,57 +15,64 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${setSelectedGood} is selected` : 'No goods selected'}
```

Comment:
There is an error in this line. You are using `setSelectedGood` directly in the template string, which is a function. You should use `selectedGood` instead to display the name of the selected good.